src/neuro_flow/batch_runner.py

```diff
@@ -258,8 +258,13 @@ class _TmpData:
                 # Reusing image ref between bakes introduces
                 # race condition anyway, so we can safely use it
                 # as remote context dir name
+                img_part = image.ref.replace(":", "/")
+                while "//" in img_part:
+                    img_part = img_part.replace("//", "/")
```

Comment:
Is there an opportunity to extract this logic into a method in `image.ref`? I am not sure this one is common enough, but we can still have a dedicated name for this. wdyt?

Reply:
`image.ref` here is `str` already, so added the dedicated method into the `image` itself, wdyt?